runtime/queries/javascript/injections.scm

```diff
@@ -9,6 +9,14 @@
   ]
   arguments: (template_string) @injection.content)
 
+; Parse the contents of gql template literals
+
+((call_expression
+   function: (identifier) @ignore
+   arguments: (template_string) @injection.content)
+ (#eq? @ignore "gql")
+ (#set! injection.language "graphql"))
```

Comment:
```suggestion
((call_expression
   function: (identifier) @_template_function_name
   arguments: (template_string) @injection.content)
 (#eq? @_template_function_name "gql")
 (#set! injection.language "graphql"))
```

re: pickfire's comment. I think this reads better and looks similar to some other injections we have like rust's regex injections

`@ignore` has special meaning when it comes to tags queries so it probably doesn't make sense to use it in highlight/injection queries